test/embedded-script-external/embedded-script-external.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Author: Chmouel Boudjnah <chmouel@chmouel.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import argparse
+import io
+import os.path
+import re
+import sys
+import typing
+
+from ruamel.yaml import YAML
+
+REGEXP = r"^#include\s*([^$]*)"
+
+
+def replace(filename: str) -> typing.List:
```

Comment:
Eventually we should have unit tests 🙏

Reply:
Yep I added some, i didn't bother because we don't have ways to run them yet 😅

I usually do pytest but i guess [python builtin unittest](https://docs.python.org/3/library/unittest.html) should be enough for this